test/api/v01/with_solver_test.rb

```diff
@@ -74,4 +74,33 @@ def test_returned_graph
     end
     delete_completed_job @job_id, api_key: 'ortools' if @job_id
   end
+
+  def test_filter_intermediate_solution
+    # verify that intermediate solution are correctly filtered
```

Comment:
As convention in ruby, the first letter of a comment with multiple words should start with an uppercase
```suggestion
    # Verify that intermediate solution are correctly filtered
```